File: cppProject/contacto.cpp
#include "contacto.h"

contacto::contacto(const std::string& NewNombre,const std::string& NewApodo,const std::string& NewEdad,
				   const std::string& NewCumple,const std::string& NewCel,const std::string& NewTel,const std::string& Correo){
	Nombre = NewNombre;
	Apodo = NewApodo;
	Edad = NewEdad;
	Cumple = NewCumple;
	Cel = NewCel;
	Tel = NewTel;
	Correo = NewCorreo;
}

// obtener valores de atributos
inline std::string contacto::getNombre()const{return Nombre;}

inline std::string contacto::getApodo()const{return Apodo;}

inline std::string contacto::getEdad()const{return Edad;}

inline std::string contacto::getCumple()const{return Cumple;}

inline std::string contacto::getCel()const{return Cel;}

inline std::string contacto::getTel()const{return Tel;}
	
inline std::string contacto::getCorreo()const{return Apodo;}

//establecer valores de atributos
inline void contacto::setNombre(const std::string& NewNombre){Nombre = NewNombre;}

inline void contacto::setApodo(const std::string& NewApodo){Apodo = NewApodo;}

inline void contacto::setEdad(const std::string& NewEdad){Edad = NewEdad;}

inline void contacto::setCumple(const std::string& NewCumple){Cumple = NewCumple;}

inline void contacto::setCel(const std::string& NewCel){Cel = NewCel;}

inline void contacto::setTel(const std::string& NewTel){Tel = NewTel;}

inline void contacto::setCorreo(const std::string& NewCorreo){Correo = NewCorreo;}

File: pythonProject/contacto.py
class contacto():	
	def __init__(self,NewNombre,NewApodo='null',NewEdad='null',
				 NewCumple='null',NewCel='null',NewTel='null',NewCorreo='null'):		
		self.Nombre = NewNombre
		self.Apodo = NewApodo
		self.Edad = NewEdad
		self.Cumple = NewCumple
		self.Cel = NewCel
		self.Tel = NewTel
		self.Correo = NewCorreo		
	# getters(metodos para obtener atributos)	
	def getNombre(self):
		return self.Nombre
	def getApodo(self):
		return self.Apodo
	def getEdad(self):
		return self.Edad
	def getCumple(self):
		return self.Cumple
	def getCel(self):
		return self.Cel
	def getTel(self):
		return self.Tel
	def getCorreo(self):
		return self.Correo		
	# setters(metodos para establecer atributos)
	def setNombre(self,NewNombre):
		self.Nombre = NewNombre
	def setApodo(self,NewApodo):
		self.Apodo = NewApodo
	def setEdad(self,NewEdad):
		self.Edad = NewEdad
	def setCumple(self,NewCumple):
		self.Cumple = NewCumple
	def setCel(self,NewCel):
		self.Cel = NewCel
	def setTel(self,NewTel):
		self.Tel = NewTel
	def setCorreo(self,NewCorreo):
		self.Correo = NewCorreo